ibm/mas_devops/roles/cp4d_wds/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: "Fail if mas_instance_id is not provided"
```

Comment:
mas_instance_id and related vars should be optional, if they are not set then the role doesn't generate the MAS cfg template.  See other roles (e.g. db2) for example code handling this

Reply:
update it